vividus-plugin-web-app/src/test/java/org/vividus/selenium/BrowserWindowSizeListenerTests.java

```diff
@@ -39,45 +40,35 @@
     @Mock private IWebDriverManager webDriverManager;
     @Mock private IWebDriverProvider webDriverProvider;
     @Mock private IBrowserWindowSizeProvider browserWindowSizeProvider;
+    @Mock private IBrowserWindowSizeManager browserWindowSizeManager;
     @InjectMocks private BrowserWindowSizeListener browserWindowSizeListener;
 
+    @SuppressWarnings("removal")
     @Test
     void shouldResizeWindowWithDesiredBrowserSizeOnCreate()
     {
         when(webDriverManager.isElectronApp()).thenReturn(false);
         when(webDriverManager.isMobile()).thenReturn(false);
-        WebDriver webDriver = mock(WebDriver.class);
-        when(webDriverProvider.get()).thenReturn(webDriver);
-        BrowserWindowSize browserWindowSize = mock(BrowserWindowSize.class);
-        Options options = mock(Options.class);
-        when(webDriver.manage()).thenReturn(options);
-        Window window = mock(Window.class);
-        when(options.window()).thenReturn(window);
-        Dimension dimension = mock(Dimension.class);
-        when(browserWindowSize.toDimension()).thenReturn(dimension);
+        BrowserWindowSize browserWindowSize = new BrowserWindowSize(1440, 900);
         boolean remoteExecution = true;
         when(webDriverProvider.isRemoteExecution()).thenReturn(remoteExecution);
-        when(browserWindowSizeProvider.getBrowserWindowSize(remoteExecution)).thenReturn(browserWindowSize);
+        when(browserWindowSizeProvider.getBrowserWindowSizeFromMeta(remoteExecution)).thenReturn(browserWindowSize);
         browserWindowSizeListener.onWebDriverCreate(new WebDriverCreateEvent(mock(WebDriver.class)));
-        verify(window).setSize(dimension);
+        verify(browserWindowSizeManager).resizeBrowserWindow(eq(browserWindowSize),
+                argThat(arg -> arg instanceof Consumer));
```

Comment:
not enough to check in this way, it's better to invoke the consumer and then verify the result

Reply:
the same for all other similar tests